modules/sbo-deploying-a-postgresql-database-operator-power-z.adoc

```diff
@@ -63,7 +65,8 @@ rules:
       - list
 EOD
 ----
-<1> The Operator image. 
+<1>  Specify `legacy` or `restricted`. The default is `legacy`. In a future {product-title} release, it is planned that the default value will be `restricted`. If your catalog cannot run with `restricted` permissions, it is recommended that you manually set this field to `legacy`.
```

Comment:
Based on the fact that the operator registry image refs refer to 4.8 and 4.9 tags below, should this specify that `legacy` must be used?

I'm unfamiliar with those images, so this might be an incorrect assumption.

Reply:
I was a little unsure about modifying an example from outside the Operator Framework. I will pull this content and reach out the writer for SBO after GA.